src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```
@@ -15,18 +16,22 @@
     left: 50%;
   }
 
-  padding: 15px;
-  border-radius: 0 0 50px 50px;
+  padding: $announcement-container-padding;
+  border-radius: 0 0 20px 20px;
   text-align: center;
 
   .close-button {
     cursor: pointer;
     position: absolute;
-    top: 10px;
+    top: 15px;
     right: 15px;
     :hover {
       color: var(--primary-color);
       text-decoration: underline;
     }
   }
 }
+
+.announcement-message {
+  margin: 0px 30px;
+}
```

At the top of the file with the other variables I would probably add something like:

```scss
$close-icon-width = 20px;
$message-margin = $spacing + $close-icon-width + 5px;
```

Then here, we could have:

```suggestion
.announcement-message {
  margin: 0 $message-margin;
}
```

Further reading:

- see reference links for SCSS math/calculations/etc in https://github.com/sparkletown/sparkle/pull/1184#discussion_r578884512